packages/modules/src/modules/banners/designableBanner/stories/DesignableBanner.stories.tsx

```diff
@@ -13,6 +14,72 @@ export default {
     args: props,
 } as Meta;
 
+// Unsafe - do not use outside of storybook
```

Comment:
What do you think of providing this context in the method name rather than a comment? `unsafeHexColourFromString` or similar? Harder to lose that information that way IMO! Alternatively there's a [version of this function](https://github.com/guardian/support-admin-console/blob/9d8b771d07173cbb990648261823864de2797b30/public/src/utils/bannerDesigns.ts#L20) which throws an error in SAC, could be worth using that?

Reply:
👍 I've reused your `stringToHexColour`. I've pasted it into both places as we probably still don't want to be using this outside of storybook/tests